src/elements/topographic-profile.ts

```diff
@@ -0,0 +1,138 @@
+import {LitElementI18n} from '../i18n.js';
+import {html} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import i18next from 'i18next';
+import draggable from './draggable';
+import ToolboxStore from '../store/toolbox';
+import {cartesianToLv95} from '../projection';
+import * as d3 from 'd3';
```

Comment:
I think d3 is not big if you only import the parts you need. Idem in package.json.

Reply:
Done in https://github.com/swissgeol/ngm/pull/752/commits/c98999c0583bd034d2635f40584bb7121841a94f